packages/cli/tests/images/build.js

```diff
@@ -11,33 +11,32 @@ exports.default = Object.assign({}, common, {
 	'assets/icons/favicon-32x32.png': 1487,
 	'assets/icons/mstile-150x150.png': 9050,
 	'assets/favicon.ico': 15086,
-	'ssr-build/ssr-bundle.aaacf.css': 1281,
-	'ssr-build/ssr-bundle.aaacf.css.map': 2070,
+	'ssr-build/ssr-bundle.89e23.css': 1281,
+	'ssr-build/ssr-bundle.89e23.css.map': 2070,
 	'ssr-build/ssr-bundle.js': 11937,
 	'ssr-build/ssr-bundle.js.map': 32557,
 	'ssr-build/asset-manifest.json': 178,
-	'bundle.2da73.css': 901,
-	'bundle.44866.js': 21429,
-	'bundle.44866.js.map': 111801,
+	'bundle.259c5.css': 901,
+	'bundle.fbf55.js': 21429,
+	'bundle.fbf55.js.map': 111801,
 	'favicon.ico': 15086,
 	'index.html': 2034,
 	'manifest.json': 455,
 	'preact_prerender_data.json': 11,
 	'push-manifest.json': 450,
 	'asset-manifest.json': 1074,
-	'route-home.chunk.bcb8a.css': 58,
-	'route-home.chunk.3cec8.js': 327,
-	'route-home.chunk.3cec8.js.map': 483,
-	'route-profile.chunk.6dd80.css': 61,
-	'route-profile.chunk.ddf94.js': 3514,
-	'route-profile.chunk.ddf94.js.map': 15454,
+	'route-home.chunk.df08e.css': 58,
+	'route-home.chunk.85051.js': 327,
+	'route-home.chunk.85051.js.map': 483,
+	'route-profile.chunk.7bcae.css': 61,
+	'route-profile.chunk.6d480.js': 3514,
+	'route-profile.chunk.6d480.js.map': 15454,
 });
 exports['default-esm'] = Object.assign({}, exports.default, {
 	'bundle.*.esm.js': 21135,
 	'bundle.*.esm.js.map': 111771,
 	'polyfills.*.esm.js': 5721,
 	'polyfills.*.esm.js.map': 21633,
-	'route-home.chunk.3cec8.js.map': 483,
```

Comment:
Silly copy/paste error that snuck in.

Reply:
should be a wildcard?

Reply:
The next line is the (correct) wildcard match:

https://github.com/preactjs/preact-cli/blob/49bba048d3c33ab490b45009957d27f8014848c5/packages/cli/tests/images/build.js#L40-L41

This was a non-esm entry that somehow snuck into the esm file list, only jumped out as the file hash changed